src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -5,18 +5,26 @@ import { FORMS as RemediationForms } from '../../ion/RemediationConstants';
 const Body = BaseForm.extend({
   title() {
     if (this.options.appState.getCurrentViewState().href.endsWith('idp/idx/enroll/update')) {
-      return loc('oie.primaryauth.submit', 'login');
+      const attributes = this.options.appState.attributes;
+      if (attributes.uiDisplay && attributes.uiDisplay.label) {
```

Comment:
is there ever a chance that `attributes` could be undefined? It would throw an error on 9 if it was.
Maybe do `attributes && attributes.uiDisplay && attributes.uiDisplay.label` ? Same with line 20

Reply:
instead you can do this `attributes?.uiDisplay?.label`

Reply:
updated as suggested. Thanks!